include/yaml-cpp/node/convert.h

```diff
@@ -241,6 +242,35 @@ struct convert<std::list<T> > {
   }
 };
 
+// std::array
+template <typename T, std::size_t S>
+struct convert<std::array<T, S> > {
+  static Node encode(const std::array<T, S>& rhs) {
+    Node node(NodeType::Sequence);
+    for (typename std::array<T, S>::const_iterator it = rhs.begin();
+        it != rhs.end(); ++it)
+      node.push_back(*it);
+    return node;
+  }
+
+  static bool decode(const Node& node, std::array<T, S>& rhs) {
+    if (!node.IsSequence())
+      return false;
```

Comment:
This should probably also fail to decode if the node has the wrong size, right? I wouldn't expect to parse [1, 2] as an array of length 1 or 3.